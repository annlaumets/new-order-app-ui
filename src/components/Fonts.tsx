import { Global } from '@emotion/react'

const Fonts = () => {
  return (
    <Global
      styles={`
      @font-face {
        font-family: 'vectrex';
        font-style: normal;
        font-weight: 700;
        font-display: swap;
        src: url('/fonts/vectrex/bold/Vectrex-Bold.woff') format('woff'), url('/fonts/vectrex/bold/Vectrex-Bold.ttf') format('ttf');
        unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
      }

      @font-face {
        font-family: 'america';
        font-style: normal;
        font-weight: 400;
        font-display: swap;
        src: url('/fonts/gt-america-mono/regular/GT-America-Mono-Regular.woff') format('woff'), url('/fonts/gt-america-mono/regular/GT-America-Mono-Regular') format('ttf');
        unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
      }
      `}
    />
  )
}

export default Fonts
