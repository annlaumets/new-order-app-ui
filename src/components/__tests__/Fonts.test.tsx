import { render } from '@testing-library/react'

import Fonts from 'components/Fonts'

describe('Fonts.tsx component', () => {
  const { container } = render(<Fonts />)

  test('match to snapshot', () => {
    expect(container).toMatchSnapshot()
  })
})
