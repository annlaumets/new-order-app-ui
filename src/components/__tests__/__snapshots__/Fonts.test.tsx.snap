// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fonts.tsx component match to snapshot 1`] = `<div />`;
